app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,10 @@
 
 @import 'variables';
 
+.red-700-border-2px-width{
```

Comment:
For a good semantic name, I'd suggest naming CSS classes in a way that describes what they're _for_, rather than what they _do_. You're using this one for an "invalid data" state, so you could call it `.invalid-input` or `.failed-validation` or something similar.

Also as a general rule: when adding new CSS, add it to the most specific file and append to the bottom. `application.scss` is the least specific file; it's just general application-wide styling. If this is only used on posts, you could append it to `posts.scss`, or if it's for comments, to `comments.scss`. (It doesn't actually matter if you use it elsewhere - all CSS is available everywhere, not just specific files in specific places, so just pick whatever it's _primarily_ for).

Reply:
I directly used it in application.scss cause it is needed it lots of pages. users, comments, posts that's why I thought all of them call the class from application.scss